pages/info/info.py

```diff
@@ -7,49 +7,151 @@
 
 layout = dbc.Container(
     [
-        dcc.Markdown(
-            """
-
-            ### Defintions
-            -------
-            __Contribution__
-                > Definition for item here.
-
-            __Contributor__
-                > Definition for item here.
-
-            __Active Contributor__
-                > Definition for item here.
-
-            __Drifting Contributor__
-                > Definition for item here.
-
-            __Fly-By Contributor__
-                > Definition for item here.
-
-            __Repeat Contributor__
-                > Definition for item here.
-
-            __Staleness__
-                > Definition for item here.
-
-            *__Note:__ These definitions are specifically for the data that is being used to populate this graph. It is not indictive of the
-            overall philosophy on open source communities.*
-
-            ### Page Community Health Lens
-            -------
-            Explaination of the grouping of visualizations on each page
-
-            ### Plotly Graph Functionality
-            -------
-
-            Plotly graphs have a mode bar if you hover over the top of the title.
-
-            If you want to reset the view of a graph with customization options, toggle one of the options to reset the view.
-
-
-        """
-        )
+        html.H3("Definitions"),
+        html.Hr(),
+        html.B("Contributor"),
+        html.Blockquote(
+            [
+                html.P(
+                    """
+                    An individual who has shared some software artifact or resource with an open source project.
```

Comment:
For these definitions (atleast priorly) the idea was to give a "metrics definition" of what exactly is being counted on a graph where we say "contributor" (contribution is an easier example as there is a list of actions and contributor is one who does said actions). Im open to having the more general definitions, but think it is important to have both in that case. A qualitative and qualitative section might be a good approach to that

Reply:
Okay definitely, that's more useful- I'll just list everything we count.

Reply:
That would work for me. I would just want to make sure to include specific verbiage about the differences between a general definition and a "metrics" or count definition. Imo the most important part of these definitions is to specify what we are counting in our different visualizations

Reply:
I'm not sure I totally understand what you're saying. Could you give an example of what you'd like to see included as well, and where?

Reply:
Instead of the "examples" describing it as "Contributions Measured" (or some type of wording that specifies what is counted in 8Knot). Then for the list it being each one individually listed (Open pull request, Close pull request, merge pull request...) or could be similar to what you have but changing the order for legibility to something like "Pull Requests: open, review, comment, close, merge"

Reply:
and then specifying somewhere that an individual is counted as a contributor when one of the measure actions is completed